lms/djangoapps/teams/static/teams/templates/edit-team.underscore

```diff
@@ -0,0 +1,33 @@
+<div class="create-team wrapper-msg is-incontext urgency-low warning is-hidden">
+    <div class="msg">
+        <div class="msg-content">
+            <h3 class="title"><%- gettext("Oops!") %></h3>
```

Comment:
@muhammad-ammar I think @clrux is saying that the h3 is required (Chris, pls correct me if I'm wrong). So I suggested some text to replace "Oops!". However, I wasn't sure how that error heading is used. My suggested text is only good if that h3 applies to various "create team" errors.